File: ex3_nn.py
from lib import *

data=io.loadmat('ex3data1.mat')
x=data['X']
x=AddOnes(x)
y=data['y']

data=io.loadmat('ex3weights.mat')
t1_trans=data['Theta1']
t2_trans=data['Theta2']
t=append(t1_trans.T,t2_trans.T)
n=[400,25,10]
lamb=0

p=NnPredict(t,n,x,y)
Print2([p])
print('MATLAB: 97.5%')

File: ex3_nn.r
source('D:/sync/ml/ml-lib/lib.r')
library('R.matlab')

data=readMat('ex3data1.mat')
x=data$X
x=AddOnes(x)
y=data$y

data=readMat('ex3weights.mat')
t1=data$Theta1
t2=data$Theta2
t=append(t(t1),t(t2))
n=c(400,25,10)
lamb=0

p=NnPred(t,n,x,y)
print(p)
## MATLAB: 97.5%
